tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -454,25 +465,27 @@ public function userHasCreatedAPublicLinkShareWithSettings(string $user, ?TableN
 	/**
 	 * @When /^the user creates a public link share using the sharing API with settings$/
 	 *
-	 * @param TableNode|null $body
+	 * @param TableNode $body
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function theUserCreatesAPublicLinkShareWithSettings(?TableNode $body):void {
-		$this->userCreatesAPublicLinkShareWithSettings($this->currentUser, $body);
+	public function theUserCreatesAPublicLinkShareWithSettings(TableNode $body):void {
+		$this->setResponse($this->createPublicLinkShare($this->currentUser, $body));
+		$this->pushToLastStatusCodesArrays();
 	}
 
 	/**
 	 * @Given /^the user has created a public link share with settings$/
 	 *
-	 * @param TableNode|null $body
+	 * @param TableNode $body
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function theUserHasCreatedAPublicLinkShareWithSettings(?TableNode $body):void {
-		$this->theUserCreatesAPublicLinkShareWithSettings($body);
+	public function theUserHasCreatedAPublicLinkShareWithSettings(TableNode $body):void {
+		$this->setResponse($this->createPublicLinkShare($this->currentUser, $body));
+		$this->pushToLastStatusCodesArrays();
```

Comment:
```suggestion
		$response = $this->createPublicLinkShare($this->currentUser, $body);
```

Reply:
This step definition is not used anywhere. Can i just delete it?